### ISSUE TITLE
Issue when formatting tick labels for log scale plots

### ISSUE THREAD

**Issue by ndawe** [piccolomo/plotext#90]
Great library! Just hit the problem below when creating a log-log plot that seems to be due to how the tick labels are constructed.

Here's a minimal example to reproduce the issue:

```python
import plotext as plt

plt.xscale("log")
plt.yscale("log")
plt.plot(
    [3.66070161e-08, 4.70771327e-07, 6.13046022e-07, 9.78958201e-01,
    9.97156618e-01, 1.00000000e+00],
    [5.60983465e-12, 5.07770834e-11, 6.79463166e-11, 5.34865283e-01,
    8.23871623e-01, 9.99762833e-01]
)
plt.show()
```

This causes:

```
  File "...site-packages/plotext/_monitor.py", line 1153, in <listcomp>
    labels = [el[: el.index('.') + d + 2] for el in labels]
ValueError: substring not found
```

The  value of `labels` at that point is:

```
['1e-11', '4.2e-10', '3.157e-08', '2.36823e-06', '0.00017765659999999998', '0.013327207709999999', '0.9997628329999999']
```

and there is no decimal point in the first label. `distinguishing_digit()` might also be returning too many digits in this case.

**Comment by piccolomo** [comment]
Great Issue report, thanks a lot. I will have a look at it. Kind of resting a bit after latest upload (it was a big work).

**Comment by nelbren** [comment]
**Quick fix:**
1)  Replace the line: 

```
    labels = [str(ut.round(el, d + 1)) for el in ticks]
```
2) For this new line:

```
    labels = [f"{ut.round(el, d + 1):f}" for el in ticks]
```

Regards! 🙂

**Comment by piccolomo** [comment]
cool, i will come back to this in following weeks. i slowly creating next minor version. if you think it works feel free to go ahead with a pull request. for now thanks for the issue and solution.
Savy

**Comment by piccolomo** [comment]
Hello @ndawe and @nelbren ,

I have solved the issue in the new [version 5.1](https://github.com/piccolomo/plotext), available only on GitHub for now, using:
```console
pip install git+https://github.com/piccolomo/plotext
```
Any feedback or double test is appreciated.
Thanks and all the best,
Savino
